test/Land/Model/constant_energy_model.jl

```diff
@@ -0,0 +1,64 @@
+using MPI
+using OrderedCollections
+using StaticArrays
+
+using CLIMAParameters
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+using ClimateMachine
+using ClimateMachine.Land
+using ClimateMachine.Mesh.Topologies
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.DGMethods
+using ClimateMachine.DGMethods.NumericalFluxes
+using ClimateMachine.DGMethods: BalanceLaw, LocalGeometry
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.VariableTemplates
+using ClimateMachine.SingleStackUtils
+
+FT = Float64;
+ClimateMachine.init(; disable_gpu = true);
+
+const clima_dir = dirname(dirname(pathof(ClimateMachine)));
+
+# Load some helper functions for plotting
+# include(joinpath(clima_dir, "docs", "plothelpers.jl"));
```

Comment:
Let's remove this

Reply:
removed this file from tracking as it isnt being used